Cargo.toml

```diff
@@ -38,14 +38,17 @@ lazy_static = { version = "1.0.2", optional = true }
 thiserror = { version = "1.0.31", optional = true }
 smallvec = { version = "1.0", optional = true }
 
+# https://github.com/open-telemetry/opentelemetry-rust/issues/2165
+tonic = { version = "0.12.3", optional = true }
+
 [dev-dependencies]
 async-trait = "0.1.56"
 criterion = { version = "0.5.1", default-features = false, features = ["html_reports"] }
-opentelemetry = { version = "0.25", features = ["trace", "metrics"] }
-opentelemetry_sdk = { version = "0.25", default-features = false, features = ["trace", "rt-tokio"] }
-opentelemetry-stdout = { version = "0.25", features = ["trace", "metrics"] }
-opentelemetry-otlp = { version = "0.25", features = ["metrics"] }
-opentelemetry-semantic-conventions = "0.25"
+opentelemetry = { version = "0.26", features = ["trace", "metrics"] }
+opentelemetry_sdk = { version = "0.26", default-features = false, features = ["trace", "rt-tokio"] }
+opentelemetry-stdout = { version = "0.26", features = ["trace", "metrics"] }
+opentelemetry-otlp = { version = "0.26", features = ["metrics"] }
+opentelemetry-semantic-conventions = { version = "0.26", features = ["semconv_experimental"] }
```

Comment:
semantic-conventions crate [introduce `semconv_experimental` feature](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-semantic-conventions/CHANGELOG.md#changed)

Reply:
And why should we enable that here? Do we depend on it in this crate?

Reply:
In the [otlp example](https://github.com/tokio-rs/tracing-opentelemetry/blob/ebc0a9209c2c7d3e60212dea7b460becca4915b1/examples/opentelemetry-otlp.rs#L25), a `Resource` was created using semantic conventions. I believe that removing `DEPLOYMENT_ENVIRONMENT_NAME` from the example won't affect its functionality, so if it can be removed, enabling the feature will no longer be necessary.

Reply:
Ahh, this is only for the dev-dependencies. I guess that's fine.